tests/ssl-opt.sh

```diff
@@ -166,7 +166,7 @@ requires_config_disabled() {
 
 requires_config_value_at_least() {
     NAME="$1"
-    DEF_VAL=$( grep ".*#define.*MBEDTLS_SSL_DTLS_MAX_BUFFERING" ../include/mbedtls/config.h |
+    DEF_VAL=$( grep ".*#define.*${NAME}" ../include/mbedtls/config.h |
```

Comment:
I would go a step further and move value extraction to a separate function.

Reply:
Addressed in https://github.com/ARMmbed/mbedtls/pull/1988/commits/7c48dd11dbee505e8bab82b2d5941cd3c190243e.